### DISCUSSION TITLE
Data for value mapping from datasource

### DISCUSSION THREAD

**Discussion by PhilippDehner** [grafana/grafana#39127] [General]
It would be nice, if there would be a possibility to use queries for the value mapping.

Currently I've to set the mapping in every panel, but if the shown values and/or from the query change, I have to change every mapping manually.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
